Editor/Resources/EditorWindow/Pages/GameLiftPluginBucketPopup.uxml

```diff
@@ -0,0 +1,13 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" engine="UnityEngine.UIElements" editor="UnityEditor.UIElements" noNamespaceSchemaLocation="../../../../../UIElementsSchema/UIElements.xsd" editor-extension-mode="False">
+    <Style src="project://database/Packages/com.amazonaws.gamelift/Editor/Resources/EditorWindow/Pages/GameLiftPluginBucketPopup.uss?fileID=7433441132597879392&amp;guid=5ec89f4b8cb28084b8cca20cda5c26ff&amp;type=3#GameLiftPluginBucketPopup" />
```

Comment:
Is it possible to move the styling from these .uxml files into the .uss files since those seem to be explicitly meant for styling? These are being bloated and difficult to read due to the styling information.

Reply:
Yep, we have a separate fork that will be pulled in soon that will contain all the inline styling into separate style sheets.